driver/src/price_finding/error.rs

```diff
@@ -36,6 +36,12 @@ impl From<&str> for PriceFindingError {
         PriceFindingError::new(error, ErrorKind::Unknown)
     }
 }
+
+impl From<(u16, u128)> for PriceFindingError {
+    fn from(_x: (u16, u128)) -> Self {
+        PriceFindingError::new("Some shit.", ErrorKind::Unknown)
```

Comment:
Is this accidentally left around? Also the conversion from a price pair to a `PriceFindingError` does not seem obvious to me.

Reply:
Ya, this was just something I was trying that didn't pan out and is not used anywhere.